### PR TITLE
chore: Lint improvements

### PR DESCRIPTION
**Pull request by stackchain**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -17,6 +17,7 @@ import {StorageProvider} from './Storage'
 enableScreens()
 
 if (Platform.OS === 'android') {
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
   if (UIManager.setLayoutAnimationEnabledExperimental) {
```
Comment (wolverineks) [inline src/App.tsx]: ?
`if (!!UIManager.setLayoutAnimationEnabledExperimental) {`

File: src/legacy/utils.ts [hunk 2]
```diff
@@ -398,3 +398,23 @@ export const ignoreConcurrentAsyncHandler = <Props, T, R>(
...
+ *
+ * @param value The string to inspect
+ * @return {boolean} Returns true if the string is empty, else false.
+ *
+ * @example isEmptyString('') returns true
+ * @example isEmptyString(' ') returns false
+ * @example isEmptyString(null) returns true
+ * @example isEmptyString(undefined) returns true
+ */
+export function isEmptyString(value: string | null | undefined): value is '' | null | undefined {
```
Comment (wolverineks) [inline src/legacy/utils.ts:418]: given the name `isEmptyString`, this function should validate that the value is of type `string` and has a length of 0
`null` and `undefined` are not strings
what is this function supposed to do?

File: src/legacy/utils.ts [hunk 3]
```diff
@@ -398,3 +398,23 @@ export const ignoreConcurrentAsyncHandler = <Props, T, R>(
...
+  return isEmpty(value)
```
Comment (wolverineks) [inline src/legacy/utils.ts:419]: have you tested this?

File: src/App.tsx (hunk 1 above)
Comment (stackchain) [reply src/App.tsx]: it will conflict with the no-extra-bool check, but I can check against null

File: src/legacy/utils.ts (hunk 2 above)
Comment (stackchain) [reply src/legacy/utils.ts:418]: It's just a temporary wrapper for lodash `isEmpty` until the safeguard PR is merged. Accepting naming suggestions.

File: src/BiometricAuth/BiometricAuthScreen.tsx [hunk 4]
```diff
@@ -73,7 +73,7 @@ export const BiometricAuthScreen = () => {
     <FingerprintScreenBase
       onGoBack={cancelScanning}
       headings={[strings.headings1, strings.headings2]}
-      subHeadings={route.params?.instructions || undefined}
+      subHeadings={route.params?.instructions ?? undefined}
```
Comment (wolverineks) [inline src/BiometricAuth/BiometricAuthScreen.tsx]: what type is `instructions`?
can this be changed to...
`route.params?.instructions ?? undefined` -> `route.params?.instructions`

Comment (stackchain) [reply src/BiometricAuth/BiometricAuthScreen.tsx]: @wolverineks, whenever is possible, please make it as git suggestions so I can commit it directly.

**Review by banklesss — APPROVED** [review]

**Pull request merged**